src/main/java/org/broadinstitute/consent/http/models/User.java

```diff
@@ -318,4 +319,12 @@ public String toString() {
         return new Gson().toJson(this);
     }
 
+    public static boolean hasUserRole(User user, UserRoles role) {
```

Comment:
Can't this just take itself?, i.e. `hasUserRole(UserRoles role) { check this.getRoles(), etc. };` It would assume that `user` isn't null so you'd want to ensure that they are non-null in the callers.

Reply:
Additionally, it would be cleaner to have it be non-static so an existing `user` could call this method.